templates/zerver/help/import-from-gitter.md

```diff
@@ -1,105 +1,127 @@
-`<subdomain>` with the desired subdomain.
+1. To import into an organization hosted on the root domain
+   (`EXTERNAL_HOST`) of the Zulip installation, run the following
+   commands.
 
-```
-./manage.py import <subdomain> converted_gitter_data
-```
+{!import-self-hosted-server-tips.md!}
 
-{!import-login.md!}
+    ```
+    cd /home/zulip/deployments/current
+    ./scripts/stop-server
+    ./manage.py convert_gitter_data /tmp/gitter_data.json --output /tmp/converted_gitter_data
+    ./manage.py import '' /tmp/converted_gitter_data
+    ./scripts/start-server
+    ```
 
-## Create organization owners
+    Alternatively, to import into a custom subdomain, run:
 
-The [Gitter API][gitter-api-user-data] don't contain data on which
-users are administrators of the Gitter channel.  As a result, all
-Gitter users are imported into Zulip as normal users.  You can follow
-the Zulip documentation on
-[making a user an organization owner from the terminal][grant-admin-access]
-to mark the appropriate users as organization owners.
+    ```
+    cd /home/zulip/deployments/current
+    ./scripts/stop-server
+    ./manage.py convert_gitter_data /tmp/gitter_data.json --output /tmp/converted_gitter_data
+    ./manage.py import <subdomain> /tmp/converted_gitter_data
+    ./scripts/start-server
+    ```
 
-[grant-admin-access]: https://zulip.readthedocs.io/en/latest/production/management-commands.html#grant-administrator-access)
-[gitter-api-user-data]: https://developer.gitter.im/docs/user-resource
+1. Follow [step 4](https://zulip.readthedocs.io/en/stable/production/install.html#step-4-configure-and-use)
+   of the guide for [installing a new Zulip
+   server](https://zulip.readthedocs.io/en/stable/production/install.html).
+
+{end_tabs}
+
+#### Import details
+
+Whether you are using Zulip Cloud or self-hosting Zulip, here are a few notes to
+keep in mind about the import process:
+
+- [Gitter's export tool](https://github.com/minrk/archive-gitter) does not export
+  workspace settings, so you will need to [configure the settings for your Zulip
+  organization](/help/customize-organization-settings). This includes settings
+  like [email visibility](/help/restrict-visibility-of-email-addresses),
+  [message editing permissions](/help/configure-message-editing-and-deletion#configure-message-editing-and-deletion_1),
+  and [how users can join your organization](/help/restrict-account-creation).
 
-## Caveats
+- Gitter's export tool does not export user settings, so users in your organization
+  may want to [customize their account settings](/help/getting-started-with-zulip).
 
-- The [Gitter data export tool](https://github.com/minrk/archive-gitter)
-  doesn't support exporting private Gitter channels.
+- The [Gitter API][gitter-api-user-data] doesn't contain data on which
+  users are administrators of a Gitter channel.  As a result, all
+  Gitter users are imported into Zulip as normal users.  You can follow
```

Comment:
Maybe we should also split it into two bullets? How about:

>- The [Gitter API](gitter-api-user-data) doesn't contain data on which users are administrators of a Gitter channel.  As a result, all Gitter users are imported into Zulip as [members](/help/roles-and-permissions). Organization owners and administrators can [change a user's role](/help/change-a-users-role).
>
>- If you [import into Zulip Cloud](#import-your-data-into-zulip), you'll need to specify the user whose account will have the owner role. If you self-host, you can follow the Zulip documentation on [making a user an organization owner from the terminal](grant-admin-access) to mark the appropriate users as organization owners.

Reply:
That looks pretty good! Maybe something like this for the Zulip Cloud part, so that it's not confusing how you specify it?

> If you [import into Zulip Cloud](https://github.com/zulip/zulip/pull/23668#import-your-data-into-zulip), you will specify the user whose account will have the owner role when you request the import.